src/osg/FrameBufferObject.cpp

```diff
@@ -21,6 +21,7 @@
 #include <osg/Texture2DMultisample>
 #include <osg/Texture3D>
 #include <osg/Texture2DArray>
+#include <osg/Texture2DMultisampleArray>
```

Comment:
Missing change to flushDeletedRenderBuffers() the GLExtensions::Get() function is false in makOsg for 3.4

Reply:
Also what happened to flushDeletedFrameBufferObjects?